src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * A task type that the chatting bot can create.
+ */
+public class Deadlines extends Task {
+
+    private LocalDate date = null;
+    private LocalTime time = null;
+
+    /**
+     * The constructor of this class.
+     *
+     * @param name
+     * @param sTime
+     */
+    public Deadlines(String name, String sTime) throws DukeException {
+        super(name);
+        DateTimeFormatter dtf = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+        DateTimeFormatter df = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Perhaps you could use more descriptive variable names here rather than dtf or df?